src/components/CreateMenu.js

```diff
@@ -95,5 +113,5 @@ class CreateMenu extends PureComponent {
 
 CreateMenu.propTypes = propTypes;
 CreateMenu.defaultProps = defaultProps;
-CreateMenu.displayName = 'CreateMenu';
```

Comment:
NAB; Just wondering, why did you remove this? I see most other components have this still 🤔

Reply:
`displayName` is only required for functional components, not class components 👍

Reply:
Although now that I'm saying that maybe it would be simpler to just always have it... 🤔
Anyways for now let's stick with the established pattern of having a `displayName` for functional components and no `displayName` for class components.

Reply:
Now I'm seeing that it is mentioned [here](https://github.com/Expensify/Expensify.cash/blob/431b60d09f4e41c7532707454625dbe2c3c22c21/STYLE.md#stateless-component-style) to only add for functional components, so good call :D

Reply:
Yeah, in the original PR I had commented about this, and the part I was unclear on is if you need a display name for pure components.

The only purpose of the display name is so that the react dev tools is useful. So I would say, remove the display name property, and if it still has a proper name in the react dev tools (eg. `CreateMenu`) then it's fine to remove it. (again, I'm just not sure when it comes to `PureComponents`.

Reply:
We can check! IIRC it only makes a difference on mobile too